File: android-sdk/fancall-sdk/src/main/kotlin/com/fancall/sdk/FanCallClient.kt
package com.fancall.sdk

/**
 * Thin client to the FanCall backend. No VoIP/media — the actual call is a
 * normal PSTN call placed by the telecom operator. Fleshed out in the
 * Android SDK plan.
 */
class FanCallClient(private val baseUrl: String) {
    fun version(): String = "0.1.0"
}

File: ios-sdk/Sources/FanCallSDK/FanCallClient.swift
import Foundation

/// Thin client to the FanCall backend. No VoIP/media — the actual call is a
/// normal PSTN call placed by the telecom operator. Fleshed out in the iOS SDK plan.
public struct FanCallClient {
    private let baseURL: String

    public init(baseURL: String) {
        self.baseURL = baseURL
    }

    public func version() -> String { "0.1.0" }
}
